pkg/cmd/server/api/install/install.go

```diff
@@ -12,6 +12,7 @@ import (
 	_ "github.com/openshift/origin/pkg/build/admission/defaults/api/install"
 	_ "github.com/openshift/origin/pkg/build/admission/overrides/api/install"
 	_ "github.com/openshift/origin/pkg/project/admission/requestlimit/api/install"
+	_ "github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride/api/install"
```

Comment:
On Tue, Feb 16, 2016 at 9:15 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/api/install/install.go
> <https://github.com/openshift/origin/pull/6901#discussion_r53014883>:
>
> > +	_ "github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride/api/install"
>
> these were moved to pkg/cmd/server/api/install/install.go
>

I followed suit and put mine in the same place as the other admission
plugins... did I miss something?